commands/test.js

```diff
@@ -257,38 +330,183 @@ module.exports = {
+        ${taskParams["systemComponentName"]}`);
       }
       if (testTypes.includes(TestType.Security)) {
         log.debug("Commencing security tests");
-        shell.cd(workdir);
-        await exec(`${shellDir}/test/security-node.sh ${taskParams["systemComponentName"]} ${version} ${JSON.stringify(taskParams["asocRepoUrl"])} ${taskParams["asocRepoUser"]} ${taskParams["asocRepoPassword"]} ${taskParams["asocAppId"]} ${taskParams["asocLoginKeyId"]} ${taskParams["asocLoginSecret"]} ${taskParams["asocClientCli"]} ${taskParams["asocJavaRuntime"]} ${shellDir} ${testdir}`);
+        await exec(
+          `${shellDir}/test/security-node.sh \
+          ${taskParams["systemComponentName"]} \
+          ${version} \
+          ${JSON.stringify(taskParams["asocRepoUrl"])} \
+          ${taskParams["asocRepoUser"]} \
+          ${taskParams["asocRepoPassword"]} \
+          ${taskParams["asocAppId"]} \
+          ${taskParams["asocLoginKeyId"]} \
+          ${taskParams["asocLoginSecret"]} \
+          ${taskParams["asocClientCli"]} \
+          ${taskParams["asocJavaRuntime"]} \
+          ${shellDir} \
+          ${testdir}`
+        );
       }
       if (testTypes.includes(TestType.SeleniumNative)) {
         log.debug("Commencing automated Selenium native tests");
-        shell.cd(workdir);
-        await exec(`${shellDir}/test/selenium-native.sh ${taskParams["systemComponentName"]} ${version} ${taskParams["saucelabsApiKey"]} ${taskParams["saucelabsApiUser"]} ${JSON.stringify(taskParams["saucelabsApiUrl"])} ${taskParams["browserName"]} ${taskParams["browserVersion"]} ${taskParams["platformType"]} ${taskParams["platformVersion"]} ${taskParams["webTestsFolder"]} ${taskParams["gitUser"]} ${taskParams["gitPassword"]} ${shellDir} ${testdir}`);
+        await exec(
+          `${shellDir}/test/selenium-native.sh \
+          ${taskParams["systemComponentName"]} ${version} \
+          ${taskParams["saucelabsApiKey"]} \
+          ${taskParams["saucelabsApiUser"]} ${JSON.stringify(taskParams["saucelabsApiUrl"])} \
+          ${taskParams["browserName"]} \
+          ${taskParams["browserVersion"]} ${taskParams["platformType"]} \
+          ${taskParams["platformVersion"]} \
+          ${taskParams["webTestsFolder"]} \
+          ${taskParams["gitUser"]} \
+          ${taskParams["gitPassword"]} \
+          ${shellDir} \
+          ${testdir}`
+        );
       }
       if (testTypes.includes(TestType.SeleniumCustom)) {
-        log.debug("Custom Selenium testing type not supported for NodeJS");
+        log.debug("Custom Selenium testing type not supported for Node.js");
       }
     } catch (e) {
       log.err("  Error encountered. Code: " + e.code + ", Message:", e.message);
       process.exit(1);
     } finally {
       await exec(shellDir + "/common/footer.sh");
-      log.debug("Finished Boomerang CICD NodeJS test activity");
+      log.debug("Finished Boomerang CICD Node.js test activity");
+    }
+  },
+  async npm() {
```

Comment:
Very similar to what is done for `nodejs`, except removed support for selenium related tests.